packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -1366,7 +1366,7 @@ void main() {
             }
             return Material(
               child: MergeSemantics(
-                child: ListTile(
+                child: CupertinoListTile(
```

Comment:
Tests are failing because of this 1 change.

Reply:
I'm unsure what's the right fix

Reply:
But issue mentioned in PR is only about example file.

Reply:
Oh, I see. I can revert the changes in this file for now.